### PR TITLE
Add v1.11 releasing blog

### PR DESCRIPTION
**Pull request by XiShanYongYe-Chang**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by XiShanYongYe-Chang** [comment]
```log
[ERROR] Client bundle compiled with errors therefore further build is impossible.
```

https://app.netlify.com/sites/karmada/deploys/66e40b243ca5160008399b5c

Hi @samzong, can you help determine what this error is about? I tried to add the image to the md file and resize and position the image

**Comment by XiShanYongYe-Chang** [comment]
I couldn't get a neat list of contributor avatars, and it didn't seem to be profitable to spend a lot of time on it, so I replaced it with an image.

/cc @samzong @windsonsea @RainbowMango

**Review by Arhell — APPROVED** [review]

**Review by windsonsea — COMMENTED** [review]
some nits

File: blog/2024-09-13-karmada-v1.11/karmada-v1.11.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+![](./img/rollout-03.png)
+
+From the example above, we can see that by using the cross-cluster rolling upgrade capability of federated workloads, the new version of the workload can be rolled out cluster by cluster, and precise control can be achieved.
+
+Additionally, this feature can also be applied to other scenarios:
+
+- As developers, situations where resources are frequently updated can arise when the Karmada control plane competes with member clusters for resource control. Suspending the process of synchronizing resources to member clusters will help quickly identify the issue.
+
+## Enhancements to Karmadactl Capabilities and Improved Operations Experience
```
[the remark this answers is not in the document]
Comment (XiShanYongYe-Chang) [reply blog/2024-09-13-karmada-v1.11/karmada-v1.11.md:81]: How about shortening the title to `Enhancements to karmadactl`

[the remark this answers is not in the document]
Comment (windsonsea) [reply blog/2024-09-13-karmada-v1.11/karmada-v1.11.md]: Looks good.

**Comment by windsonsea** [comment]
Thanks
/lgtm

File: blog/2024-09-13-karmada-v1.11/karmada-v1.11.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
+# Karmada v1.11 version released! New cross-cluster rolling upgrade capability for workload!
+
+Karmada is an open multi-cloud and multi-cluster container orchestration engine designed to help users deploy and operate business applications in a multi-cloud environment. With its compatibility with the native Kubernetes API, Karmada can smoothly migrate single-cluster workloads while still maintaining coordination with the surrounding Kubernetes ecosystem tools.
+
+This version includes the following new features:
+- Supports cross-cluster rolling upgrades of federated workloads, making the user's version release process more flexible and controllable.
+- karmadactl has added multiple operational capabilities, providing a unique multi-cluster operational experience.
+- It provides standardized generation semantics for federated workloads, enabling CD execution in one step.
+Karmada Operator supports custom CRD download strategies, making offline deployment more flexible.
+- Overview of New Features
```
Comment (RainbowMango) [inline blog/2024-09-13-karmada-v1.11/karmada-v1.11.md:10]:
```suggestion
- It provides standardized generation semantics for federated workloads, enabling CD execution in one step.
- Karmada Operator supports custom CRD download strategies, making offline deployment more flexible.
```

File: blog/2024-09-13-karmada-v1.11/karmada-v1.11.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+- GroupVersion: apps/v1 Kind: Deployment, DaemonSet, StatefulSet
+- GroupVersion: apps.kruise.io/v1alpha1 Kind: CloneSet, DaemonSet
+- GroupVersion: apps.kruise.io/v1beta1 Kind: StatefulSet
+- GroupVersion: helm.toolkit.fluxcd.io/v2beta1 Kind: HelmRelease
+- GroupVersion: kustomize.toolkit.fluxcd.io/v1 Kind: Kustomization
+- GroupVersion: source.toolkit.fluxcd.io/v1 Kind: GitRepository
+- GroupVersion: source.toolkit.fluxcd.io/v1beta2 Kind: Bucket, HelmChart, HelmRepository, OCIRepository
+
+If you need to adapt more resources (including CRDs), you can provide feedback to the Karmada community or extend using the Resource Interpreter.
```
Comment (RainbowMango) [inline blog/2024-09-13-karmada-v1.11/karmada-v1.11.md:113]:
```suggestion
If you need to adapt more resources (including CRDs), you can provide feedback to the Karmada community or extend using [the Resource Interpreter](https://karmada.io/docs/userguide/globalview/customizing-resource-interpreter).
```

File: blog/2024-09-13-karmada-v1.11/karmada-v1.11.md (hunk 2 above)
Comment (XiShanYongYe-Chang) [reply blog/2024-09-13-karmada-v1.11/karmada-v1.11.md]: thx, updated.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**